src/tactic/ui/tools/trigger_wdg.py

```diff
@@ -185,6 +185,9 @@ def get_display(my):
         search = Search("sthpw/notification")
         if my.mode == 'pipeline':
             search.add_filter("process", my.process)
+            if my.process_sobj:
```

Comment:
you need to wrap being and then or around the add_filter if you are doing or. It's recommended. Look at the else condition for an example.. begin is needed when u use or.